strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/domain/PypiPackageInfo.java

```diff
@@ -98,4 +114,35 @@ public String getPlatform()
         return platform;
     }
 
+    public enum SupportedMetadataVersionEnum
+    {
+        VERSION_1_0("1.0"),
+        VERSION_1_1("1.1"),
+        VERSION_1_2("1.2"),
+        VERSION_2_1("2.1");
+
+        private String version;
+
+        public String getVersionString(){
+            return version;
+        }
+
+        public static SupportedMetadataVersionEnum getVersionEnum(String version) throws IllegalArgumentException
+        {
+
+            for (SupportedMetadataVersionEnum metadataVersionEnum : SupportedMetadataVersionEnum.values())
+            {
+                if (metadataVersionEnum.getVersionString().equals(version))
+                {
+                    return metadataVersionEnum;
+                }
+            }
+            throw new IllegalArgumentException("Unsupported Metadata version: " + version);
+        }
+
+        SupportedMetadataVersionEnum(String version){
```

Comment:
Cosmetic: Would you mind following our [coding convention](https://strongbox.github.io/developer-guide/coding-convention.html)? Braces are on new lines. Thanks! :)